packages/headless/src/api/platform-client.ts

```diff
@@ -134,6 +127,32 @@ function coveoCloudURL<E extends PlatformEnvironment>(
   return `https://${subdomain}${urlEnv}${urlRegion}.cloud.coveo.com`;
 }
 
+/**
+ * Returns the unique endpoint(s) for a given organization identifier.
+ * @param orgId The organization identifier.
+ * @param env Optional. The environment (prod, hipaa, staging, dev) that the organization belongs to. Defaults to `prod`.
+ * @returns
```

Comment:
Won't there always be three endpoints? Namely, platform, analytics and search?
```suggestion
 * Returns the unique platform, analytics and search endpoints for a given organization identifier.
 * @param orgId The organization identifier.
 * @param env Optional. The environment (prod, hipaa, staging, dev) that the organization belongs to. Defaults to `prod`.
 * @returns The unique platform, analytics and search endpoints for a given organization identifier.
```

Reply:
I prefer not to be specific with the object returned.

It's possible that we will need to return different endpoints down the line. 
We can add new keys without breaking the contract.

Reply:
> For now I'm thinking we can live with comments in this file only.

Yep. Also, for anyone using TS, it's all going to be safely typed